docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -521,7 +521,9 @@ Status Codes:
 
 `GET /containers/(id)/stats`
 
-This endpoint returns a live stream of a container's resource usage statistics.
+This endpoint will returns a live stream of a running container's resource usage statistics.
+It will attach to any created container, but will only send resource usage information
+when the container is running.
```

Comment:
Eliminate conditional, make active voice: It attaches to any created container, but only sends resource usage information when the container is running.